contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/Box.vue

```diff
@@ -100,22 +112,42 @@
 <style lang="scss" scoped>
 
   .box {
-    padding: 8px;
-    color: v-bind('boxTextColor');
+    padding: 12px 16px;
```

Comment:
Here, the box should have a vertical and horizontal padding of 10px to match the specs styles.